data/api/client-server/message_pagination.yaml

```diff
@@ -49,6 +49,7 @@ paths:
         - in: query
           type: string
           name: from
+          x-changedInMatrixVersion: "1.3"
```

Comment:
this doesn't seem to be showing anything in the rendered spec.

Generally we need to describe *how* it changed, not just that it changed.

Reply:
So just prepend `**Since v1.3:**` to the new paragraph in the description, or something else?

Reply:
have a look at `data/api/client-server/definitions/client_event_without_room_id.yaml` for an example of how to use `x-changedInMatrixVersion` correctly

Reply:
hrm, it still doesn't seem to be rendering. We'll have to dig into what's wrong with the templates :/

Reply:
#1011 should fix this.